src/plugins/media.ts

```diff
@@ -225,6 +225,21 @@ export class MediaPlugin {
   })
   stopRecord(): void { }
 
+  /**
+   * Pauses recording
+   */
+  @CordovaInstance({
+    sync: true
+  })
```

Comment:
hmm the docs seem to be out of date since Android is also supported, which i've verified on my device

https://github.com/apache/cordova-plugin-media/blob/88a69305eb7f2e7c03481f951a1d349f2a8b4ad0/src/android/AudioHandler.java#L123